mypy/sitepkgs.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import print_function
+# NOTE: This file must remain compatible with Python 2
+
+
+from distutils.sysconfig import get_python_lib
+import site
+from typing import List
```

Comment:
Should you assume that the python 2 installation has `typing`?

Reply:
If you are type checking Python 2 code I hope you would have typing installed. Just to be safe I can do:

```
MYPY = False
if MYPY:
    from typing import List
```